docs/event-sources/README.md

```diff
@@ -2,19 +2,25 @@
 
 **Event sources turn any API into an event stream. They can also turn any event stream into an API**.
 
-Sources collect data from services like Github, Stripe, the bitcoin blockchain, RSS feeds, and more. They emit new events produced by the service, which you can consume using Pipedream's APIs.
+Event sources run on Pipedream's infrastructure and collect data from services like Github, Stripe, the bitcoin blockchain, RSS feeds, and more. They emit new events produced by the service, which you can inspect in the UI or consume using real-time or batch APIs.
```

Comment:
I'd link to the REST and SSE API where you say "real-time" and "batch"

Reply:
done